handbook/engineering/campaigns/goals.md

```diff
@@ -65,3 +56,11 @@ We maintain a [directionally correct roadmap](https://sourcegraph.productboard.c
 <img src="./roadmap-updated-2021-02-12.png" alt="Campaigns short term roadmap updated 2021-02-12" />
 
 *Updated 2021-02-12*
+
+# How we built those goals and milestones
+We generally think about campaigns features in terms of 4 themes:
+
+- expand the realm where campaigns can run
+- create many changesets easily
+- manage and track many changesets easily
+- get changesets merged faster
```

Comment:
At least as we've talked about these in the past, we usually combined "manage" and "merge" as one item, and included as a fourth item "performance and robustness", so something like:

```suggestion
1. create many changesets easily
1. track, manage, and merge many changesets easily
1. expand the realm where campaigns can run
1. keep campaigns robust and performant
```